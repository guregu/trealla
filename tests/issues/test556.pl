main :-
	X = (-𝒶𝒶 + 𝒶𝒶),
	writeq(X), nl.
