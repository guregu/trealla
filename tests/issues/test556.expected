-𝒶𝒶+𝒶𝒶
